openreview/conference/templates/programchairWebfield.js

```diff
@@ -429,6 +446,47 @@ var displaySortPanel = function(container, sortOptions, sortResults) {
   });
 };
 
+var addTagsToPaperSummaryCell = function(data, pcAssignmentTagInvitations) {
+
+  if (pcAssignmentTagInvitations.length) {
```

Comment:
I think condition this should be `pcAssignmentTagInvitations && pcAssignmentTagInvitations.length` since `getPcAssignmentTagInvitations()` can return undefined.

Reply:
I also think this is a situation where you want to check the inverse of the condition and return, rather than wrapping the whole function body in an if.

Reply:
Agreed. I have made changes for this.